Sources/AppcuesKit/Presentation/Public/Plugins/AppcuesTraitError.swift

```diff
@@ -16,9 +16,16 @@ public struct AppcuesTraitError: Error, CustomStringConvertible {
     /// This value will be logged to Appcues Studio.
     public var description: String
 
+    /// When set, this value can indicate a number of milliseconds after which re-application
+    /// of traits can be attempted, to try to auto-recover from this error.
+    public var retryMilliseconds: Int?
```

Comment:
I'm not sure about exposing this publicly. I'm not opposed, but it'd be good to think through a bit more before committing to that.

Reply:
I'm not sure about exposing any Trait code publicly :) but yes, I figured might as well give the option, in a non-breaking optional add-on way here, for any custom traits. It could certainly be kept internal though for now.

Reply:
rebased this one on `main` - no longer any public API question here 😏